cirq/google/engine/engine.py

```diff
@@ -316,6 +388,55 @@ def create_program(
         return engine_program.EngineProgram(self.project_id, new_program_id,
                                             self.context, new_program)
 
+    def create_batch_program(
+            self,
+            programs: List['cirq.Circuit'],
+            program_id: Optional[str] = None,
+            gate_set: serializable_gate_set.SerializableGateSet = None,
+            description: Optional[str] = None,
+            labels: Optional[Dict[str, str]] = None,
+    ) -> engine_program.EngineProgram:
+        """Wraps a list of Circuits into a BatchProgram for the Quantum Engine.
+
+        Args:
+            programs: The Circuits to execute within a batch.
+            program_id: A user-provided identifier for the program. This must be
+                unique within the Google Cloud project being used. If this
+                parameter is not provided, a random id of the format
+                'prog-################YYMMDD' will be generated, where # is
+                alphanumeric and YYMMDD is the current year, month, and day.
+            gate_set: The gate set used to serialize the circuit. The gate set
+                must be supported by the selected processor
+            description: An optional description to set on the program.
+            labels: Optional set of labels to set on the program.
+
+        Returns:
+            A EngineProgram for the newly created program.
+        """
+        gate_set = gate_set or gate_sets.XMON
+
+        if not program_id:
+            program_id = _make_random_id('prog-')
+
+        code = qtypes.any_pb2.Any()
```

Comment:
Implemented the first suggestion.

I hesitate to mess with the client too much, since it is partially generated by the gapic tools, so, if we regenerate the library, we don't want to mess with any customizations to the greatest extent possible.